_scripts/linux.just

```diff
@@ -33,12 +40,18 @@ install-deps:
     fi
 
     if [[ ! -d "${LIBCLANG_PATH}" ]]; then
-        sudo apt-get install -y libclang-13-dev
+        if [ "{{AptGetExists}}" == "true" ]; then
+            sudo apt-get install -y libclang-13-dev
+        fi
     fi
 
     pushd {{ExtDir}}
 
     if [ ! -d "{{ExtDir}}/{{QtVersion}}" ]; then
+        echo "Setting up python venv in {{ExtDir}}/venv"
+        python -m venv "{{ExtDir}}/venv"
```

Comment:
I think this should be `python3 -m venv`.
Python2 just doesn't refuse to die unfortunately